docs/pages/installation/kubernetes.md

```diff
@@ -4,51 +4,62 @@ title: Kubernetes
 permalink: installation/platforms/kubernetes
 ---
 
-# Quick Start with Kubernetes
+## Quick Start with Kubernetes
+
+To set up and run Meshery on Kubernetes:
+
+- <a href="#helm">Use Helm and set up a Kubernetes cluster</a>
+
+- <a href="#k8s">Run Meshery on existing Kubernetes cluster</a>
   
-<a name="helm"></a>
-## Using Helm
+<a name="helm">
 
-### Helm v3
+#### <b> Using Helm</b>
```

Comment:
Looks like we may need to centrally update the `<h3>` style, if we need `<b>` (which is deprecated) in many places.

Reply:
Update -
- Removed all instances of duplicate anchors and any mixing of html and markdown. 
- Sticking to markdown for docs from now unless unavoidable. 
- Not centrally updating the h3 tag for now because I'm not sure of the changes it would cause throughout the docs.